tests/jenkins/TestRunNonSecBenchmarkTestScript.groovy

```diff
@@ -0,0 +1,187 @@
+            c -> lib.jenkins.BuildManifest.new(readYaml(file: 'tests/jenkins/data/opensearch-1.3.0-non-security-bundle.yml'))
+        })
+        helper.registerAllowedMethod('parameterizedCron', [String], null)
+        helper.registerAllowedMethod("cfnDescribe", [Map])
+        helper.registerAllowedMethod("cfnDelete", [Map])
+        binding.setVariable('AGENT_LABEL', 'Jenkins-Agent-AL2-X64-M52xlarge-Docker-Host-Benchmark-Test')
+        binding.setVariable('ARCHITECTURE', 'x64')
+        binding.setVariable('ARTIFACT_BUCKET_NAME', 'test_bucket')
+        binding.setVariable('ARTIFACT_DOWNLOAD_ROLE_NAME', 'Dummy_Download_Role')
+        binding.setVariable('AWS_ACCOUNT_PUBLIC', 'dummy_account')
+        binding.setVariable('env', ['BUILD_NUMBER': '307'])
+        binding.setVariable('BUILD_NUMBER', '307')
+        binding.setVariable('BUILD_URL', 'test://artifact.url')
+        binding.setVariable('BUILD_ID', '1234')
+        binding.setVariable('BUNDLE_MANIFEST', 'tests/jenkins/data/opensearch-1.3.0-non-security-bundle.yml')
+        binding.setVariable('BUNDLE_MANIFEST_URL', 'test://artifact.url')
+        binding.setVariable('GITHUB_BOT_TOKEN_NAME', 'bot_token_name')
+        binding.setVariable('GITHUB_USER', 'test_user')
+        binding.setVariable('GITHUB_TOKEN', 'test_token')
+        binding.setVariable('HAS_SECURITY', 'true')
+        binding.setVariable('SINGLE_NODE_CLUSTER', 'false')
+        binding.setVariable('MIN_DISTRIBUTION', 'false')
+        binding.setVariable('USE_50_PERCENT_HEAP', 'true')
+        binding.setVariable('SUFFIX', '1234')
+        binding.setVariable('MANAGER_NODE_COUNT', '3')
+        binding.setVariable('DATA_NODE_COUNT', '3')
+        binding.setVariable('USER_TAGS', 'run-type:test')
+        binding.setVariable('WORKLOAD_PARAMS', '')
+        binding.setVariable('ADDITIONAL_CONFIG', '')
+        binding.setVariable('CLIENT_NODE_COUNT', '')
+        binding.setVariable('INGEST_NODE_COUNT', '')
+        binding.setVariable('ML_NODE_COUNT', '')
+        binding.setVariable('DATA_NODE_STORAGE', '100')
+        binding.setVariable('ML_NODE_STORAGE', '')
+        binding.setVariable('JVM_SYS_PROPS', '')
+        binding.setVariable('CAPTURE_NODE_STAT', 'false')
+        binding.setVariable('JOB_NAME', 'benchmark-test')
+        binding.setVariable('BENCHMARK_TEST_CONFIG_LOCATION', 'test_config')
+        binding.setVariable('PUBLIC_ARTIFACT_URL', 'test://artifact.url')
+        binding.setVariable('STAGE_NAME', 'test_stage')
+        binding.setVariable('TEST_WORKLOAD', 'nyc-taxis')
+        binding.setVariable('WEBHOOK_URL', 'test://artifact.url')
+
+        super.setUp()
+    }
+
+    @Test
+    public void testRunNonSecurityBenchmarkTestScript_verifyPipeline() {
+        super.testPipeline("jenkins/opensearch/benchmark-test.jenkinsfile",
+                "tests/jenkins/jenkinsjob-regression-files/opensearch/benchmark-test.jenkinsfile")
+    }
```

Comment:
Looks like we are adding just 1 txt file where security is set to `false` I believe we can add one more?

```
    @Test
    public void testRunSecurityBenchmarkTestScript_verifyPipeline() {
        super.testPipeline("jenkins/opensearch/benchmark-test.jenkinsfile",
                "tests/jenkins/jenkinsjob-regression-files/opensearch/benchmark-test-without-security.jenkinsfile")
    }

Reply:
Added tests for Security as well.